_posts/2021-06-08-slate-on-cloudlab.md

```diff
@@ -85,16 +83,59 @@ First, although many different Linux distributions can be used, the SLATE team r
 Second, an additional pool of floating public IP addresses must be allocated for ingress and OSG applications to work.
 The CloudLab documentation has instructions for this [here](http://docs.cloudlab.us/advanced-topics.html#%28part._dynamic-public-ip%29).*
 
-
-## Kubernetes Cluster Creation / SLATE Registration
+## Creating a Kubernetes Cluster
 
 Once our CloudLab experiment/instances have fully spun up, we can begin installing Kubernetes on the node(s).
 The SLATE team recommends that [Kubespray](https://kubespray.io/#/) be used for this.
 
 However, we need a few additional pieces of information before we begin configuring Kubespray.
 
+### Requirements
+
+This serves as an outline for setting up a SLATE Kubernetes cluster on CloudLab. For the specific commands or versions please refer to the SLATE Kubernetes Cluster Creation documentation [here](https://slateci.io/docs/cluster/automated/kubernetes-cluster-creation.html#run).
+
+What you will need to set up and run a succesful SLATE Kubernetes cluster:
+  1. a CloudLab experiment
+  2. key for CloudLab
+  3. `ansible_ssh_common_args` added to hosts.yaml
+  4. a public IP and an IP ranger for MetalLb
+  5. CertManager
+  6. approriate version for Kubernetes
+  7. appropriate Docker and Calico versions
+
+What you will *not* need to set up a SLATE Kuberentes cluster on CloudLab:
```

Comment:
This part is a little confusing. Kubernetes can handle the presence of a NAT and does use Calico to create IPs for the services and pods. Maybe see what it looks like if this set of bullets is excluded?